ang/test/mocks/data/cases-types.data.js

```diff
@@ -346,10 +346,15 @@
         /**
          * Returns a list of case types
          *
+         * @param {object} isSequential whether data should be returned in array format
          * @returns {object} a list of case types indexed by id.
          */
-        get: function () {
-          return _.clone(caseTypesMock);
+        get: function (isSequential) {
```

Comment:
Instead of having a boolean parameter why not have a different function `getSequential` ? seeing a `get(true)` call is very confusing because the `true` parameter doesn't really explains what it's doing. Does it mean it will only get active case types? Case types belonging to `cases` ? It could be anything.